### ISSUE TITLE
`is_training` for Batch Normalization in gluon

### ISSUE THREAD

**Issue by jonbakerfish** [zackchase/mxnet-the-straight-dope#492]
In [Batch Normalization from scratch](https://gluon.mxnet.io/chapter04_convolutional-neural-networks/cnn-batch-norm-scratch.html), there is a `is_training` parameter to control the behavior of Batch Normalization. However, for [Batch Normalization in gluon](https://gluon.mxnet.io/chapter04_convolutional-neural-networks/cnn-batch-norm-gluon.html), there is no such an option. How do we know it's using global moving statistics or local batch-norm in GLUON?

**Comment by patriciaaa82** [comment]
Same doubt here! Could you please clarify?
Thank you,
Patricia

**Comment by kli-nlpr** [comment]
@patriciaaa82 @jonbakerfish  I think you may find the answer here
https://gluon.mxnet.io/chapter03_deep-neural-networks/mlp-dropout-gluon.html#Accessing-is_training()-status
